src/app/services/data/trainrun-section-times.service.ts

```diff
@@ -36,7 +36,7 @@ export class TrainrunSectionTimesService {
     rightLock: false,
     travelTimeLock: false,
   };
-  private showWarningTwoLocks = false;
+  private warning = "";
```

Comment:
Could we use a union type for this field? A raw string makes this less type-safe than before.